apps/test-app/App.tsx

```diff
@@ -46,6 +49,16 @@ function loadTests({
     }
   });
 
+  describeIf(nodeTests, "Node Tests", () => {
+    for (const [suiteName, examples] of Object.entries(nodeTestsSuites)) {
+      describe(suiteName, () => {
+        for (const [exampleName, requireTest] of Object.entries(examples)) {
+          it(exampleName, requireTest);
+        }
+      });
```

Comment:
I'm not sure whether our test runner supports it, but wondering whether we could use [describe.each / test.each](https://medium.com/@missakaherath96/test-each-and-describe-each-in-jest-6bc890add93f) for better test readouts. I haven't tried before, so no worries if it's not worth it in this case.